project/Dependencies.scala

```diff
@@ -202,7 +202,7 @@ object Dependencies {
         .map("org.apache.geode" % _ % GeodeVersion) ++
       Seq(
         "com.chuusai" %% "shapeless" % "2.3.3",
-        "org.apache.logging.log4j" % "log4j-to-slf4j" % "2.12.1" % Test
+        "org.apache.logging.log4j" % "log4j-to-slf4j" % "2.16.0" % Test
```

Comment:
Right now 2.17.0 is the newest version: https://logging.apache.org/log4j/2.x/index.html

Reply:
Updated.